src/lib/moonforge/core.js

```
@@ -0,0 +1,110 @@
+    language: nav.language ?? '', hostname: loc.hostname ?? '', timestamp: unixSeconds(),
+  };
+}
+
+export async function postEvent(payload, { beacon = false } = {}) {
+  if (!state.config) return false;
+  const url = `${state.config.apiEndpoint}/api/send`;
+  const body = JSON.stringify(payload);
+  if (beacon && typeof globalThis.navigator?.sendBeacon === 'function') {
+    // sendBeacon returning false (e.g. queue full) is a soft failure — fall
+    // through to the fetch(keepalive) path so unload-time events still send.
+    try { const ok = globalThis.navigator.sendBeacon(url, new Blob([body], { type: 'application/json' })); debugLog('beacon', payload.type, ok); if (ok) return true; }
```

ESLint

[error] Replace `·const·ok·=·globalThis.navigator.sendBeacon(url,·new·Blob([body],·{·type:·'application/json'·}));·debugLog('beacon',·payload.type,·ok);·if·(ok)·return·true;·}` with `⏎······const·ok·=·globalThis.navigator.sendBeacon(⏎········url,⏎········new·Blob([body],·{·type:·"application/json"·}),⏎······);⏎······debugLog("beacon",·payload.type,·ok);⏎······if·(ok)·return·true;`

(prettier/prettier)